src/main/views/live-case-status.njk

```diff
@@ -16,7 +16,8 @@
 {% block content %}
     <h2 class="govuk-heading-l">Live hearing updates</h2>
     <h3 class="govuk-heading-m">{{ courtName }}</h3>
-    <p class=" govuk-body-lead">{{ updateDateTime }}</p>
+    <p class=" govuk-body-lead">{{ updateDate }}</p>
+    <p class=" govuk-body">Last updated as at {{ updateTime }}</p>
```

Comment:
can you extract "Last updated as at " and replace with a nunjuks place holder? it looks the same as the above, and you would place it in `resources/locales/en/live-case-status.json` create a new attr. in there with this text

Reply:
Should also be done now